typings/objection/index.d.ts

```diff
@@ -64,8 +64,8 @@ declare namespace Objection {
     // than an identity function type. <M extends typeof Model> retains the
     // model subclass type in the return value, without requiring the user
     // to type the Mixin call.
```

Comment:
This comment is obsolete and no longer matches the code below. Consider removing?

Reply:
will do